.gitignore

```diff
@@ -4,4 +4,5 @@ target/
 .idea/
 application.conf
 sqlproxy.env
-.bsp
\ No newline at end of file
+.bsp
+application.conf.qi
```

Comment:
do you have a local `application.conf.qi` that you use to test these changes? is that config documented anywhere?

Reply:
I don't actually have it anymore...but good to have it in .gitignore just so I don't have to worry about a copy of my own config file